help_to_heat/frontdoor/views.py

```diff
@@ -764,15 +797,36 @@ def build_extra_context(self, request, session_id, page_name, data, is_change_pa
         session_data = interface.api.session.get_session(session_id)
 
         address = session_data.get(address_field)
+        show_epc_update_details = session_data.get(country_field) in [country_field_wales, country_field_england]
+
         epc = session_data.get(epc_details_field)
 
         epc_band = epc.get("current-energy-rating")
+        epc_date = epc.get("lodgement-date")
+        try:
+            working_epc_date = datetime.strptime(epc_date, "%Y-%m-%d")
+            month_name = month_names[working_epc_date.month - 1]
+            gds_epc_date = working_epc_date.strftime("%-d") + " " + month_name + " " + working_epc_date.strftime("%Y")
+        except ValueError:
```

Comment:
Why would we hit this?

Reply:
If the EPC field isn't the correct format for the strptime, it would cause a value error.
So it would fall back to just showing the date as whatever the EPC says.